lib/ui/vote_persona.dart

```diff
@@ -44,6 +45,9 @@ class _VotePersonaScreenState extends State<VotePersonaScreen> {
   }
 
   Widget buildPersonaTile(String persona) {
+    final height = MediaQuery.of(context).size.height;
+    final scaleFactor = height/677;
```

Comment:
Is it 677 or 667?

Reply:
it is 667

Reply:
tested but it won't work as well
maybe we could 
fix it in the next refactor